doc/cla/individual/jbradsha.md

```diff
@@ -0,0 +1,11 @@
+United Kingdom, 2015-05-22
+
+I hereby agree to the terms of the Odoo Individual Contributor License
+Agreement v1.0.
+
+I declare that I am authorized and able to make this agreement and sign this
+declaration.
+
+Signed,
+
+John Bradshaw - jbradsha@github.com - https://github.com/jbradsha
```

Comment:
The github-generated address is actually `jbradsha@users.noreply.github.com` since you're committing through the github interface and have opted to keep your email address private. The CLA checker is currently not smart enough to handle that case.